Mindscape.Raygun4Net.NetCore.Common/RaygunClientBase.cs

```
@@ -22,7 +23,7 @@ public abstract class RaygunClientBase
       // The default timeout is 100 seconds for the HttpClient, 
       Timeout = TimeSpan.FromSeconds(30)
     };
```

One other thing that can prevent SNAT and thread exhausting is setting `MaxConnectionsPerServer` to something less than an unlimited number by default. It can be achieved by passing HttpClientHandler

```c#
new HttpClient(new HttpClientHandler
  {
      MaxConnectionsPerServer = 50
  }) {
  Timeout = TimeSpan.FromSeconds(30)
  };
```

---

We did initially look at this as a solution. 

This issue with this is that is silently drops errors from being sent, and in our testing it doesn't always solve the SNAT issue 😿

---

We actually tested this initially and found it didn't have any major contribution of reducing the number of ports taken, for example when set to 10, i was still hitting around ~2000 ports being taken in Azure and of 50k requests, only around ~4k would succeed, while leaving it defaulted it tried to use around ~3000 and ~49k requests succeeded.